records_mover/db/bigquery/load_job_config_options.py

```diff
@@ -324,13 +324,14 @@ def add_load_job_csv_config(unhandled_hints: Set[str],
     quiet_remove(unhandled_hints, 'datetimeformat')
 
     if hints.datetimeformattz in ['YYYY-MM-DD HH:MI:SSOF',
+                                  'YYYY-MM-DD HH24:MI:SSOF',
                                   'YYYY-MM-DD HH:MI:SS']:
```

Comment:
An example of where we were missing testing - YYYY-MM-DD HH24:MI:SSOF is an supported existing value, identical to `YYYY-MM-DD HH:MI:SSOF` (which was already listed in this code).